lib/Core/SpecialFunctionHandler.cpp

```diff
@@ -693,8 +693,8 @@ void SpecialFunctionHandler::handleMakeSymbolic(ExecutionState &state,
     
     const ObjectState *old = it->first.second;
     ExecutionState *s = it->second;
-    
-    if (old->readOnly) {
+
+    if (old->readOnly || isConstant(mo->allocSite)) {
```

Comment:
Yes, `readOnly` is currently used to mark external objects (with the exception of `errno`) as read-only.